main-app/tests/cssSelection/css-toggle.js

```diff
@@ -0,0 +1,27 @@
+(function () {
+    'use strict';
+    describe('Testing css-toggle', function () {
+        var constants;
+        beforeEach(function () {
+            module('Tombola.NoughtAndCrosses.cssSelection');
+            inject(function ($injector) {
+                constants = $injector.get('cssToggle');
+            });
+        });
+
+        it('Ensures the default css is main.css', function () {
+            assert.equal('main.css', constants.cssStyle);
```

Comment:
This should use the should syntax - see the chai documentation.
Sort and let me know